### DISCUSSION TITLE
AC0A6D71-A967-46D6-AC27-4C328B0CF0C3

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#201] [General]
# Planet Ten

Planet Ten is a hypothetical planet in the outer region of the Solar System. Their gravitational effects could explain the Kuiper Cliff anomaly, where the amount of KBOs suddenly drops at around 47.8 AU. If it does exist, Planet Ten is estimated to be a rocky planet (similar in size to Mars). This add-on not only adds the hypothetical into celestia, but it also gives it the name 

https://celestia.mobi/addon?item=AC0A6D71-A967-46D6-AC27-4C328B0CF0C3

**Comment by AKJUS** [comment]
Nice to join

**Comment by exocritic5457** [comment]
Someone I've heard of before, for a change? Props for simply not making it blue and yellow like all of those clickbait thumbnails, that's a sight for sore eyes.

**Reply by Zarina-5** [reply]
Well, maybe not considering that white is the brightest color with a lightness value of 100%.

**Comment by redouaneabda-oss** [comment]
Excuse me Planet 10’s bugged. Whenever i go to Planet 10 (in game) it always shows me a black sphere. I tried Ambient light, nothing. Nothing worked for me to get Planet 10’s surface shown.
